govcd/api_vcd_test.go

```diff
@@ -258,13 +258,14 @@ type TestConfig struct {
 		OvaName        string `yaml:"ovaName,omitempty"`
 	} `yaml:"cse,omitempty"`
 	SolutionAddOn struct {
-		Org           string `yaml:"org"`
-		Vdc           string `yaml:"vdc"`
-		RoutedNetwork string `yaml:"routedNetwork"`
-		ComputePolicy string `yaml:"computePolicy"`
-		StoragePolicy string `yaml:"storagePolicy"`
-		Catalog       string `yaml:"catalog"`
-		AddonImageDse string `yaml:"addonImageDse"`
+		Org           string                       `yaml:"org"`
+		Vdc           string                       `yaml:"vdc"`
+		RoutedNetwork string                       `yaml:"routedNetwork"`
+		ComputePolicy string                       `yaml:"computePolicy"`
+		StoragePolicy string                       `yaml:"storagePolicy"`
+		Catalog       string                       `yaml:"catalog"`
+		AddonImageDse string                       `yaml:"addonImageDse"`
+		DseSolutions  map[string]map[string]string `yaml:"dseSolutions,omitempty"`
```

Comment:
I think we need a comment on this line: why a map of maps?

Reply:
Good point. I tried to explain - hopefully this makes sense. The use case for such approach is even better seen in terraform code where it has the same structure and is fed into tests directly.